### PR TITLE
docs: add image_providers jsdocs

### PR DESCRIPTION
**Pull request by ST-DDT**
These won't be displayed in the api-docs yet.

### CODE REVIEW

File: src/image_providers/lorempicsum.ts [hunk 1]
```diff
@@ -62,35 +66,38 @@ export class LoremPicsum {
...
 
   /**
-   * avatar
+   * Returns a random avatar url.
+   *
+   * @example
+   * faker.internet.avatar()
+   * // 'https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/315.jpg'
    */
+  // TODO ST-DDT 2022-03-11: Deprecate this method as it is duplicate as has nothing to do with lorempicsum.
```
Comment (Shinigami92) [inline src/image_providers/lorempicsum.ts:80]:
```suggestion
  // TODO ST-DDT 2022-03-11: Deprecate this method as it is duplicate and has nothing to do with lorempicsum.
```

File: src/image_providers/lorempicsum.ts [hunk 2]
```diff
@@ -62,35 +66,38 @@ export class LoremPicsum {
...
+   * @param height The height of the image. Defaults to `480`.
+   * @param grayscale Whether to return a grayscale image. Default to `false`.
+   * @param blur The optional level of blur to apply. Supports `1` - `10`.
+   * @param seed The optional seed to use.
    */
   imageUrl(
-    width?: number,
-    height?: number,
-    grayscale?: boolean,
+    width = 640,
```
Comment (Shinigami92) [inline src/image_providers/lorempicsum.ts:95]: This slightly changes behavior, cause previously `0` would fallback to `640`

Comment (ST-DDT) [reply src/image_providers/lorempicsum.ts:95]: Should I revert this change?

Comment (ST-DDT) [reply src/image_providers/lorempicsum.ts]: Done

**Review by Shinigami92 — APPROVED** [review]

**Review by prisis — APPROVED** [review]

**Pull request merged**